spec/Section 3 -- Type System.md

```diff
@@ -1692,6 +1686,20 @@ While defining a directive, it must not reference itself directly or indirectly:
 directive @invalidExample(arg: String @invalidExample) on ARGUMENT_DEFINITION
 ```
 
+**Built-in Directives**
```

Comment:
It seems like the confusion is specifically about introspection, so additional clarity should be added to that section rather than including a new sub-section here. Currently this section seems to mostly just repeat the content from the existing `@skip`, `@include` and `@deprecated` sections. As we add additional directives over time we should avoid having too many places we need to edit to expand that list.

Reply:
I'm also concerned about "built-ins" becoming an ambiguous term here since the directives defined are not all required to be provided by a service.

I'd prefer either "The directives specified by this document" as representing the full set of non-custom directives, or something like "directives specified by this document and provided by a GraphQL service" if referring to the subset actually supported

Reply:
My implementation of schema introspection add `includeBuiltin` param for `types` and `directives` fields. It's up to the user whether he want to query builtin types/directives or not.

```graphql
type __Schema {
  description: String
  types(includeBuiltin: Boolean = false): [__Type!]!
  queryType: __Type!
  mutationType: __Type
  subscriptionType: __Type
  directives(includeBuiltin: Boolean = false): [__Directive!]!
}
````

my implementation is able to query the introspection schema itself, that's why I added those params. to avoid confusion for new users.